examples/pretrained_word_embeddings.py

```diff
@@ -105,15 +105,15 @@
 print('Preparing embedding matrix.')
 
 # prepare embedding matrix
-num_words = min(MAX_NB_WORDS, len(word_index))
+num_words = min(MAX_NB_WORDS, len(word_index) + 1)
 embedding_matrix = np.zeros((num_words, EMBEDDING_DIM))
 for word, i in word_index.items():
     if i >= MAX_NB_WORDS:
         continue
     embedding_vector = embeddings_index.get(word)
     if embedding_vector is not None:
         # words not found in embedding index will be all-zeros.
-        embedding_matrix[i] = embedding_vector
+        embedding_matrix[i - 1] = embedding_vector
```

Comment:
That's a bug, the i-th entry in the embedding matrix should correspond to the work of index i (as returned by the tokenizer).

Reply:
You haven't fixed this. This line should be reverted.